website/docs/r/kusto_eventhub_data_connection.html.markdown

```diff
@@ -107,6 +106,8 @@ The following arguments are supported:
 
 * `data_format` - (Optional) Specifies the data format of the EventHub messages. Allowed values: `APACHEAVRO`, `AVRO`, `CSV`, `JSON`, `MULTIJSON`, `ORC`, `PARQUET`, `PSV`, `RAW`, `SCSV`, `SINGLEJSON`, `SOHSV`, `TSVE`, `TSV`, `TXT`, and `W3CLOGFILE`.
 
+* `database_routing` - (Optional) Indication for database routing information from the data connection, by default only database routing information is allowed. Allowed values: `Single`, `Multi`.
```

Comment:
should we make this
```suggestion
* `database_routing_type` - (Optional) Indication for database routing information from the data connection, by default only database routing information is allowed. Allowed values: `Single`, `Multi`.
```
?

Reply:
Ok, let me change this to `database_routing_type`

Reply:
Shall we change all `database_routing` in other two PRs to `database_routing_type`? Although in azure API, this was called databaseRouting.